tensorflow/core/lib/io/cache_test.cc

```diff
@@ -0,0 +1,238 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/core/lib/io/cache.h"
+
+#include <string>
+#include <vector>
+
+#include "tensorflow/core/platform/coding.h"
```

Comment:
`third_party/tensorflow/core/lib/io/cache_test.cc:21:10: error: use of private header from outside its module: 'third_party/tensorflow/core/platform/coding.h'; `

Reply:
@byronyi can you please check this error ?

Reply:
@rthadur Fixed. Mind to kickoff CI again?